README.md

```diff
@@ -1 +1,19 @@
-# hw01
\ No newline at end of file
+### Brief Biography - Nelson O. Puc Cruz
+##### HW1
+
+I was born and raised in Chetumal, a city of around 150,000 people in the southern state of Quintana Roo, México. The first child to my parents and grandchild to theirs, my family was ecstatic over my arrival (or so I've been told), although the only advantage that came out of my earliness was to work as a private driver for my two younger siblings on later years. I studied my first years of education in schools I don't really remember the name of, until I attended high school at the _Colegio de Bachilleres Plantel Chetumal 1_. This was followed by 4 and a half years of solving equation systems, which granted me a bachelor's degree in Economics and Finance, awarded by the _Universidad de Quintana Roo_. After graduating, I worked at the state government until the rampant corruption forced me to consider a change of career (no, really). I decided to try and get a degree in something I was interested in, and after getting the Fulbright scholarship I was ready (as in, my wallet) to take the challenge. Currently I am in my 3rd quarter of the Masters of Public Policy at the University of Chicago, where I participate actively in the following student organizations:
```

Comment:
Merit: It is nice that you included all the markdown elements (bold, header, link embed, image embed, and list) in your biography and organized them well! And it's also interesting to read your personal story before your current study direction. 

Improvement: I would suggest dividing this relatively long paragraph into several short ones to elucidate your points clearer for the readers. A little spacing would help a lot in creating sections for these!